termine.html

```diff
@@ -44,14 +44,15 @@
         <div class="content" onclick="">
           <div class="standard-article" style="text-align: center;">
             <h2>Termine und Fahrerliste</h2>
+            <p>Aktuell ist keine Fahrerliste verfügbar</p>
             <img
               style="display: unset;"
-              src="images/jugendfahrplanvorschau2020.PNG"
-              alt="Vorschaubild Jugendfahrplan"
+              src="images/ttvb_info_vorschau.JPG"
```

Comment:
Das Vorschaubild liegt nämlich im Ordner "other" und nicht in "images"

Reply:
Vorschaubild nach "images" verschoben